.travis.yml

```diff
@@ -13,12 +13,7 @@ notifications:
   email:
     on_success: never
     on_failure: always
-before_install:
-- npm update
 install:
 - npm install
-before_script:
-- npm run clean
```

Comment:
May be there is no need of removing clean script ,we can add this to `package.json`:
`"clean": "rm -r dist/*"` what it does is just removes a bunch of files so you can start with a fresh working copy to start building into.